Cannot apply after io.micronaut.application plugin

**Describe the bug**
Plugin cannot be applied after the micronaut plugin (not sure if this is a micronaut plugin issue or this one)
**To Reproduce**
Checkout https://github.com/mnembrini/rest-api-generator-example/tree/error_api_configuration and refresh gradle

**Expected behavior**
The plugin is applied without errors.
If the order is changed, then it applies without issue (micronaut plugin last)

**Error**
```
A problem occurred configuring root project 'rest-api-generator'.
> Cannot change dependencies of dependency configuration ':api' after it has been included in dependency resolution.

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.ProjectConfigurationException: A problem occurred configuring root project 'rest-api-generator'.
	at org.gradle.configuration.project.LifecycleProjectEvaluator.wrapException(LifecycleProjectEvaluator.java:75)
	at org.gradle.configuration.project.LifecycleProjectEvaluator.addConfigurationFailure(LifecycleProjectEvaluator.java:68)
	at org.gradle.configuration.project.LifecycleProjectEvaluator.access$400(LifecycleProjectEvaluator.java:51)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate.run(LifecycleProjectEvaluator.java:195)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:74)
	at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:74)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$EvaluateProject.lambda$run$0(LifecycleProjectEvaluator.java:105)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.lambda$applyToMutableState$0(DefaultProjectStateRegistry.java:325)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.lambda$withProjectLock$3(DefaultProjectStateRegistry.java:385)
	at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:211)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.withProjectLock(DefaultProjectStateRegistry.java:385)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.fromMutableState(DefaultProjectStateRegistry.java:366)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.applyToMutableState(DefaultProjectStateRegistry.java:324)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$EvaluateProject.run(LifecycleProjectEvaluator.java:91)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:74)
	at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:74)
	at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:63)
	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:750)
	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:150)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.ensureConfigured(DefaultProjectStateRegistry.java:298)
	at org.gradle.execution.TaskPathProjectEvaluator.configure(TaskPathProjectEvaluator.java:41)
	at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:55)
	at org.gradle.configuration.DefaultProjectsPreparer.prepareProjects(DefaultProjectsPreparer.java:50)
	at org.gradle.configuration.BuildTreePreparingProjectsPreparer.prepareProjects(BuildTreePreparingProjectsPreparer.java:64)
	at org.gradle.configuration.BuildOperationFiringProjectsPreparer$ConfigureBuild.run(BuildOperationFiringProjectsPreparer.java:52)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:74)
	at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:74)
	at org.gradle.configuration.BuildOperationFiringProjectsPreparer.prepareProjects(BuildOperationFiringProjectsPreparer.java:40)
	at org.gradle.initialization.VintageBuildModelController.prepareProjects(VintageBuildModelController.java:97)
	at org.gradle.initialization.VintageBuildModelController.doBuildStages(VintageBuildModelController.java:77)
	at org.gradle.initialization.VintageBuildModelController.prepareToScheduleTasks(VintageBuildModelController.java:64)
	at org.gradle.internal.build.DefaultBuildLifecycleController.lambda$prepareToScheduleTasks$0(DefaultBuildLifecycleController.java:104)
	at org.gradle.internal.build.DefaultBuildLifecycleController.lambda$withModelOrThrow$5(DefaultBuildLifecycleController.java:142)
	at org.gradle.internal.build.DefaultBuildLifecycleController.withModel(DefaultBuildLifecycleController.java:157)
	at org.gradle.internal.build.DefaultBuildLifecycleController.withModelOrThrow(DefaultBuildLifecycleController.java:140)
	at org.gradle.internal.build.DefaultBuildLifecycleController.prepareToScheduleTasks(DefaultBuildLifecycleController.java:102)
	at org.gradle.internal.buildtree.DefaultBuildTreeWorkPreparer.scheduleRequestedTasks(DefaultBuildTreeWorkPreparer.java:33)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.lambda$doScheduleAndRunTasks$1(DefaultBuildTreeLifecycleController.java:83)
	at org.gradle.composite.internal.DefaultIncludedBuildTaskGraph.withNewTaskGraph(DefaultIncludedBuildTaskGraph.java:94)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.doScheduleAndRunTasks(DefaultBuildTreeLifecycleController.java:82)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.lambda$fromBuildModel$0(DefaultBuildTreeLifecycleController.java:71)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.runBuild(DefaultBuildTreeLifecycleController.java:104)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.fromBuildModel(DefaultBuildTreeLifecycleController.java:69)
	at org.gradle.tooling.internal.provider.runner.AbstractClientProvidedBuildActionRunner.runClientAction(AbstractClientProvidedBuildActionRunner.java:56)
	at org.gradle.tooling.internal.provider.runner.ClientProvidedPhasedActionRunner.run(ClientProvidedPhasedActionRunner.java:53)
	at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
	at org.gradle.internal.buildtree.ProblemReportingBuildActionRunner.run(ProblemReportingBuildActionRunner.java:47)
	at org.gradle.launcher.exec.BuildOutcomeReportingBuildActionRunner.run(BuildOutcomeReportingBuildActionRunner.java:69)
	at org.gradle.tooling.internal.provider.FileSystemWatchingBuildActionRunner.run(FileSystemWatchingBuildActionRunner.java:91)
	at org.gradle.launcher.exec.BuildCompletionNotifyingBuildActionRunner.run(BuildCompletionNotifyingBuildActionRunner.java:41)
	at org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.lambda$execute$0(RootBuildLifecycleBuildActionExecutor.java:40)
	at org.gradle.composite.internal.DefaultRootBuildState.run(DefaultRootBuildState.java:154)
	at org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.execute(RootBuildLifecycleBuildActionExecutor.java:40)
	at org.gradle.internal.buildtree.DefaultBuildTreeContext.execute(DefaultBuildTreeContext.java:40)
	at org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.lambda$execute$0(BuildTreeLifecycleBuildActionExecutor.java:56)
	at org.gradle.internal.buildtree.BuildTreeState.run(BuildTreeState.java:53)
	at org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.execute(BuildTreeLifecycleBuildActionExecutor.java:56)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionExecutor.java:61)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionExecutor.java:57)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:200)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:195)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:62)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$call$2(DefaultBuildOperationExecutor.java:79)
	at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.callWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:54)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:79)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor.execute(RunAsBuildOperationBuildActionExecutor.java:57)
	at org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.lambda$execute$0(RunAsWorkerThreadBuildActionExecutor.java:38)
	at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:211)
	at org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.execute(RunAsWorkerThreadBuildActionExecutor.java:38)
	at org.gradle.tooling.internal.provider.ContinuousBuildActionExecutor.execute(ContinuousBuildActionExecutor.java:103)
	at org.gradle.tooling.internal.provider.SubscribableBuildActionExecutor.execute(SubscribableBuildActionExecutor.java:64)
	at org.gradle.internal.session.DefaultBuildSessionContext.execute(DefaultBuildSessionContext.java:46)
	at org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.lambda$execute$0(BuildSessionLifecycleBuildActionExecuter.java:56)
	at org.gradle.internal.session.BuildSessionState.run(BuildSessionState.java:69)
	at org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.execute(BuildSessionLifecycleBuildActionExecuter.java:55)
	at org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.execute(BuildSessionLifecycleBuildActionExecuter.java:37)
	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:36)
	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:25)
	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:63)
	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:31)
	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:58)
	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:42)
	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:47)
	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:31)
	at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:65)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:39)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:29)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:35)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:78)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:75)
	at org.gradle.util.internal.Swapper.swap(Swapper.java:38)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:75)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:63)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:84)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:52)
	at org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:297)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:61)
Caused by: org.gradle.api.InvalidUserDataException: Cannot change dependencies of dependency configuration ':api' after it has been included in dependency resolution.
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.preventIllegalMutation(DefaultConfiguration.java:1281)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.validateMutation(DefaultConfiguration.java:1241)
	at org.gradle.api.internal.artifacts.configurations.DefaultConfiguration.lambda$validateMutationType$1(DefaultConfiguration.java:309)
	at org.gradle.internal.ImmutableActionSet$SingletonSet.execute(ImmutableActionSet.java:225)
	at org.gradle.api.internal.DefaultDomainObjectSet.assertMutableCollectionContents(DefaultDomainObjectSet.java:71)
	at org.gradle.api.internal.DefaultDomainObjectCollection.add(DefaultDomainObjectCollection.java:248)
	at org.gradle.api.internal.DelegatingDomainObjectSet.add(DelegatingDomainObjectSet.java:110)
	at org.gradle.api.internal.artifacts.DefaultDependencySet.add(DefaultDependencySet.java:68)
	at org.gradle.api.internal.artifacts.DefaultDependencySet.add(DefaultDependencySet.java:36)
	at org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.doAddRegularDependency(DefaultDependencyHandler.java:189)
	at org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.doAdd(DefaultDependencyHandler.java:183)
	at org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.access$100(DefaultDependencyHandler.java:74)
	at org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler$DirectDependencyAdder.add(DefaultDependencyHandler.java:429)
	at org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler$DirectDependencyAdder.add(DefaultDependencyHandler.java:424)
	at org.gradle.api.internal.artifacts.dsl.dependencies.DynamicAddDependencyMethods.tryInvokeMethod(DynamicAddDependencyMethods.java:56)
	at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.invokeMethod(BeanDynamicObject.java:504)
	at org.gradle.internal.metaobject.BeanDynamicObject.tryInvokeMethod(BeanDynamicObject.java:196)
	at org.gradle.internal.metaobject.CompositeDynamicObject.tryInvokeMethod(CompositeDynamicObject.java:98)
	at org.gradle.internal.extensibility.MixInClosurePropertiesAsMethodsDynamicObject.tryInvokeMethod(MixInClosurePropertiesAsMethodsDynamicObject.java:34)
	at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:56)
	at ch.silviowangler.gradle.restapi.RestApiPlugin$_apply_closure11$_closure12.doCall(RestApiPlugin.groovy:113)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.util.internal.ClosureBackedAction.execute(ClosureBackedAction.java:72)
	at org.gradle.util.internal.ConfigureUtil.configureTarget(ConfigureUtil.java:155)
	at org.gradle.util.internal.ConfigureUtil.configure(ConfigureUtil.java:106)
	at org.gradle.api.internal.project.DefaultProject.dependencies(DefaultProject.java:1236)
	at org.gradle.api.Project$dependencies$4.call(Unknown Source)
	at ch.silviowangler.gradle.restapi.RestApiPlugin$_apply_closure11.doCall(RestApiPlugin.groovy:110)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.configuration.internal.DefaultListenerBuildOperationDecorator$BuildOperationEmittingClosure$1.lambda$run$0(DefaultListenerBuildOperationDecorator.java:180)
	at org.gradle.configuration.internal.DefaultUserCodeApplicationContext$CurrentApplication.reapply(DefaultUserCodeApplicationContext.java:98)
	at org.gradle.configuration.internal.DefaultListenerBuildOperationDecorator$BuildOperationEmittingClosure$1.run(DefaultListenerBuildOperationDecorator.java:177)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:74)
	at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:74)
	at org.gradle.configuration.internal.DefaultListenerBuildOperationDecorator$BuildOperationEmittingClosure.doCall(DefaultListenerBuildOperationDecorator.java:174)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:41)
	at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:25)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:43)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:245)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:157)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:61)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:346)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:249)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:141)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:37)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy52.afterEvaluate(Unknown Source)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate$1.execute(LifecycleProjectEvaluator.java:187)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate$1.execute(LifecycleProjectEvaluator.java:184)
	at org.gradle.api.internal.project.DefaultProject.stepEvaluationListener(DefaultProject.java:1428)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$NotifyAfterEvaluate.run(LifecycleProjectEvaluator.java:193)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:74)
	at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:74)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$EvaluateProject.lambda$run$0(LifecycleProjectEvaluator.java:105)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.lambda$applyToMutableState$0(DefaultProjectStateRegistry.java:325)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.lambda$withProjectLock$3(DefaultProjectStateRegistry.java:385)
	at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:211)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.withProjectLock(DefaultProjectStateRegistry.java:385)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.fromMutableState(DefaultProjectStateRegistry.java:366)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.applyToMutableState(DefaultProjectStateRegistry.java:324)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$EvaluateProject.run(LifecycleProjectEvaluator.java:91)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:74)
	at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:74)
	at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:63)
	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:750)
	at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:150)
	at org.gradle.api.internal.project.DefaultProjectStateRegistry$ProjectStateImpl.ensureConfigured(DefaultProjectStateRegistry.java:298)
	at org.gradle.execution.TaskPathProjectEvaluator.configure(TaskPathProjectEvaluator.java:41)
	at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:55)
	at org.gradle.configuration.DefaultProjectsPreparer.prepareProjects(DefaultProjectsPreparer.java:50)
	at org.gradle.configuration.BuildTreePreparingProjectsPreparer.prepareProjects(BuildTreePreparingProjectsPreparer.java:64)
	at org.gradle.configuration.BuildOperationFiringProjectsPreparer$ConfigureBuild.run(BuildOperationFiringProjectsPreparer.java:52)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:74)
	at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:74)
	at org.gradle.configuration.BuildOperationFiringProjectsPreparer.prepareProjects(BuildOperationFiringProjectsPreparer.java:40)
	at org.gradle.initialization.VintageBuildModelController.prepareProjects(VintageBuildModelController.java:97)
	at org.gradle.initialization.VintageBuildModelController.doBuildStages(VintageBuildModelController.java:77)
	at org.gradle.initialization.VintageBuildModelController.prepareToScheduleTasks(VintageBuildModelController.java:64)
	at org.gradle.internal.build.DefaultBuildLifecycleController.lambda$prepareToScheduleTasks$0(DefaultBuildLifecycleController.java:104)
	at org.gradle.internal.build.DefaultBuildLifecycleController.lambda$withModelOrThrow$5(DefaultBuildLifecycleController.java:142)
	at org.gradle.internal.build.DefaultBuildLifecycleController.withModel(DefaultBuildLifecycleController.java:157)
	at org.gradle.internal.build.DefaultBuildLifecycleController.withModelOrThrow(DefaultBuildLifecycleController.java:140)
	at org.gradle.internal.build.DefaultBuildLifecycleController.prepareToScheduleTasks(DefaultBuildLifecycleController.java:102)
	at org.gradle.internal.buildtree.DefaultBuildTreeWorkPreparer.scheduleRequestedTasks(DefaultBuildTreeWorkPreparer.java:33)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.lambda$doScheduleAndRunTasks$1(DefaultBuildTreeLifecycleController.java:83)
	at org.gradle.composite.internal.DefaultIncludedBuildTaskGraph.withNewTaskGraph(DefaultIncludedBuildTaskGraph.java:94)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.doScheduleAndRunTasks(DefaultBuildTreeLifecycleController.java:82)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.lambda$fromBuildModel$0(DefaultBuildTreeLifecycleController.java:71)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.runBuild(DefaultBuildTreeLifecycleController.java:104)
	at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.fromBuildModel(DefaultBuildTreeLifecycleController.java:69)
	at org.gradle.tooling.internal.provider.runner.AbstractClientProvidedBuildActionRunner.runClientAction(AbstractClientProvidedBuildActionRunner.java:56)
	at org.gradle.tooling.internal.provider.runner.ClientProvidedPhasedActionRunner.run(ClientProvidedPhasedActionRunner.java:53)
	at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
	at org.gradle.internal.buildtree.ProblemReportingBuildActionRunner.run(ProblemReportingBuildActionRunner.java:47)
	at org.gradle.launcher.exec.BuildOutcomeReportingBuildActionRunner.run(BuildOutcomeReportingBuildActionRunner.java:69)
	at org.gradle.tooling.internal.provider.FileSystemWatchingBuildActionRunner.run(FileSystemWatchingBuildActionRunner.java:91)
	at org.gradle.launcher.exec.BuildCompletionNotifyingBuildActionRunner.run(BuildCompletionNotifyingBuildActionRunner.java:41)
	at org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.lambda$execute$0(RootBuildLifecycleBuildActionExecutor.java:40)
	at org.gradle.composite.internal.DefaultRootBuildState.run(DefaultRootBuildState.java:154)
	at org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.execute(RootBuildLifecycleBuildActionExecutor.java:40)
	at org.gradle.internal.buildtree.DefaultBuildTreeContext.execute(DefaultBuildTreeContext.java:40)
	at org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.lambda$execute$0(BuildTreeLifecycleBuildActionExecutor.java:56)
	at org.gradle.internal.buildtree.BuildTreeState.run(BuildTreeState.java:53)
	at org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.execute(BuildTreeLifecycleBuildActionExecutor.java:56)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionExecutor.java:61)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionExecutor.java:57)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:200)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:195)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:62)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$call$2(DefaultBuildOperationExecutor.java:79)
	at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.callWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:54)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:79)
	at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor.execute(RunAsBuildOperationBuildActionExecutor.java:57)
	at org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.lambda$execute$0(RunAsWorkerThreadBuildActionExecutor.java:38)
	at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:211)
	at org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.execute(RunAsWorkerThreadBuildActionExecutor.java:38)
	at org.gradle.tooling.internal.provider.ContinuousBuildActionExecutor.execute(ContinuousBuildActionExecutor.java:103)
	at org.gradle.tooling.internal.provider.SubscribableBuildActionExecutor.execute(SubscribableBuildActionExecutor.java:64)
	at org.gradle.internal.session.DefaultBuildSessionContext.execute(DefaultBuildSessionContext.java:46)
	at org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.lambda$execute$0(BuildSessionLifecycleBuildActionExecuter.java:56)
	at org.gradle.internal.session.BuildSessionState.run(BuildSessionState.java:69)
	at org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.execute(BuildSessionLifecycleBuildActionExecuter.java:55)
	at org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.execute(BuildSessionLifecycleBuildActionExecuter.java:37)
	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:36)
	at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:25)
	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:63)
	at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:31)
	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:58)
	at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:42)
	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:47)
	at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:31)
	at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:65)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:39)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:29)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:35)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:78)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:75)
	at org.gradle.util.internal.Swapper.swap(Swapper.java:38)
	at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:75)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:63)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:84)
	at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
	at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
	at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:52)
	at org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:297)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:61)
```

--- a/build.gradle
+++ b/build.gradle
@@ -6,7 +6,7 @@ plugins {
 
 subprojects { subproject ->
 
-  version = "2.3.14"
+  version = "2.3.15"
 
   apply from: "${rootDir}/gradle/java.gradle"
   apply from: "${rootDir}/gradle/groovy.gradle"

--- a/gradle-plugin/build.gradle
+++ b/gradle-plugin/build.gradle
@@ -62,12 +62,4 @@ task checkVersion(type: Test) {
 	if (!content.contains("final String pluginVersion = \"${project.version}\"")) {
 		throw new RuntimeException(msg)
 	}
-
-	if (!content.contains("implementation \"ch.silviowangler.rest:rest-api-spring:\${pluginVersion}\"")) {
-		throw new RuntimeException(msg)
-	}
-
-	if (!content.contains("implementation \"ch.silviowangler.rest:rest-api-micronaut:\${pluginVersion}\"")) {
-		throw new RuntimeException(msg)
-	}
 }

--- a/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/GenerationMode.groovy
+++ b/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/GenerationMode.groovy
@@ -49,8 +49,7 @@ enum GenerationMode {
 	/**
 	 * Generates the framework specific client implementation.
 	 */
-	CLIENT;
-
+	CLIENT
 
 	boolean isApiCodeGenerationRequired() {
 		return this != IMPLEMENTATION;

--- a/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/GeneratorUtil.groovy
+++ b/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/GeneratorUtil.groovy
@@ -133,7 +133,7 @@ class GeneratorUtil {
 		if (verb == 'Get') {
 			String resourceModelName = createResourceModelName(fileName, verb)
 			if (collection) {
-				return ParameterizedTypeName.get(ClassName.get(targetFramework.isMicronaut() ? Iterable.class : Collection.class), ClassName.get(packageName, resourceModelName))
+				return ParameterizedTypeName.get(ClassName.get(targetFramework.micronaut ? Iterable.class : Collection.class), ClassName.get(packageName, resourceModelName))
 			} else {
 				return ClassName.get(packageName, resourceModelName)
 			}

--- a/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/RestApiPlugin.groovy
+++ b/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/RestApiPlugin.groovy
@@ -42,116 +42,116 @@ import static ch.silviowangler.gradle.restapi.TargetFramework.SPRING_BOOT
  */
 class RestApiPlugin implements Plugin<Project> {
 
-  public static final String PLUGIN_ID = 'ch.silviowangler.restapi'
+	public static final String PLUGIN_ID = 'ch.silviowangler.restapi'
 
 
-  void apply(Project project) {
+	void apply(Project project) {
 
-    project.apply(plugin: 'java-library')
+		project.apply(plugin: 'java-library')
 
-    RestApiExtension extension = new RestApiExtension(project)
-    project.extensions.add('restApi', extension)
+		RestApiExtension extension = new RestApiExtension(project)
+		project.extensions.add('restApi', extension)
 
 
-    def clean = project.tasks.register('cleanRestArtifacts', CleanRestApiTask) { CleanRestApiTask t ->
-      t.group = TASK_GROUP_REST_API
-    }
-    def extract = project.tasks.register('extractSpecs', ExtractRestApiSpecsTask) { ExtractRestApiSpecsTask t ->
-      t.group = TASK_GROUP_REST_API
-    }
-    def validate = project.tasks.register('validateRestSpecs', ValidationTask) { ValidationTask t ->
-      t.group = TASK_GROUP_REST_API
-      if (ExtractRestApiSpecsTask.isConfigurationRestApiDefined(project)) {
-        t.dependsOn(extract)
-      }
-    }
-    def generate = project.tasks.register('generateRestArtifacts', GenerateRestApiTask) { GenerateRestApiTask t ->
-      t.group = TASK_GROUP_REST_API
-      t.dependsOn(validate)
-    }
-    def generateDiagrams = project.tasks.register('generateDiagrams', PlantUmlTask) { PlantUmlTask t ->
-      t.group = TASK_GROUP_REST_API
-    }
-    def generateAsciiDocs = project.tasks.register('generateAsciiDocs', GenerateRestApiAsciiDocTask) { GenerateRestApiAsciiDocTask t ->
-      t.group = TASK_GROUP_REST_API
-    }
+		def clean = project.tasks.register('cleanRestArtifacts', CleanRestApiTask) { CleanRestApiTask t ->
+			t.group = TASK_GROUP_REST_API
+		}
+		def extract = project.tasks.register('extractSpecs', ExtractRestApiSpecsTask) { ExtractRestApiSpecsTask t ->
+			t.group = TASK_GROUP_REST_API
+		}
+		def validate = project.tasks.register('validateRestSpecs', ValidationTask) { ValidationTask t ->
+			t.group = TASK_GROUP_REST_API
+			if (ExtractRestApiSpecsTask.isConfigurationRestApiDefined(project)) {
+				t.dependsOn(extract)
+			}
+		}
+		def generate = project.tasks.register('generateRestArtifacts', GenerateRestApiTask) { GenerateRestApiTask t ->
+			t.group = TASK_GROUP_REST_API
+			t.dependsOn(validate)
+		}
+		def generateDiagrams = project.tasks.register('generateDiagrams', PlantUmlTask) { PlantUmlTask t ->
+			t.group = TASK_GROUP_REST_API
+		}
+		def generateAsciiDocs = project.tasks.register('generateAsciiDocs', GenerateRestApiAsciiDocTask) { GenerateRestApiAsciiDocTask t ->
+			t.group = TASK_GROUP_REST_API
+		}
 
-    project.tasks.named('clean').configure {
-      dependsOn(clean)
-    }
+		project.tasks.named('clean').configure {
+			dependsOn(clean)
+		}
 
-    project.tasks.named('compileJava').configure {
-      dependsOn(generate)
-      options.encoding = 'UTF-8'
-    }
+		project.tasks.named('compileJava').configure {
+			dependsOn(generate)
+			options.encoding = 'UTF-8'
+		}
 
-    project.tasks.named('compileTestJava').configure {
-      options.encoding = 'UTF-8'
-    }
+		project.tasks.named('compileTestJava').configure {
+			options.encoding = 'UTF-8'
+		}
 
-    project.sourceSets.main.java.srcDir { project.restApi.generatorOutput }
+		project.sourceSets.main.java.srcDir { project.restApi.generatorOutput }
 
-    Configuration restApiSpecification = project.configurations.findByName(CONFIGURATION_REST_API)
+		Configuration restApiSpecification = project.configurations.findByName(CONFIGURATION_REST_API)
 
-    if (!restApiSpecification) {
-      project.configurations.create(CONFIGURATION_REST_API)
-    }
+		if (!restApiSpecification) {
+			project.configurations.create(CONFIGURATION_REST_API)
+		}
 
-    final String springVersion = "5.2.4.RELEASE"
-    final String pluginVersion = "2.3.15"
-    final String libPhoneNumberVersion = "8.11.5"
+		final String springVersion = "5.2.4.RELEASE"
+		final String pluginVersion = "2.3.15"
+		final String libPhoneNumberVersion = "8.11.5"
 
-    final List<String> deps = [
-        "javax.annotation:javax.annotation-api:1.3.2",
-        "ch.silviowangler.rest:rest-model:${pluginVersion}",
-        "javax.money:money-api:1.0.3",
-        "javax.validation:validation-api:2.0.1.Final"
-    ]
+		final List<String> deps = [
+			"javax.annotation:javax.annotation-api:1.3.2",
+			"ch.silviowangler.rest:rest-model:${pluginVersion}",
+			"javax.money:money-api:1.0.3",
+			"javax.validation:validation-api:2.0.1.Final"
+		]
 
-    def api = project.configurations.named("api")
-    def implementation = project.configurations.named("implementation")
-    def compileOnly = project.configurations.named("compileOnly")
+		def api = project.configurations.named("api")
+		def implementation = project.configurations.named("implementation")
+		def compileOnly = project.configurations.named("compileOnly")
 
-    if (extension.generationMode.isApiCodeGenerationRequired() || extension.generationMode.isClientCodeGenerationRequired()) {
-      api.configure { a ->
-        a.withDependencies {
-          deps.each { dep -> it.add(project.dependencies.create(dep)) }
-        }
-      }
-    } else {
-      implementation.configure { impl ->
-        impl.withDependencies {
-          deps.each { dep -> it.add(project.dependencies.create(dep)) }
-        }
-      }
-    }
+		if (extension.generationMode.isApiCodeGenerationRequired() || extension.generationMode.isClientCodeGenerationRequired()) {
+			api.configure { a ->
+				a.withDependencies {
+					deps.each { dep -> it.add(project.dependencies.create(dep)) }
+				}
+			}
+		} else {
+			implementation.configure { impl ->
+				impl.withDependencies {
+					deps.each { dep -> it.add(project.dependencies.create(dep)) }
+				}
+			}
+		}
 
-    implementation.configure { impl ->
-      impl.withDependencies {
-        it.add(project.dependencies.create("com.googlecode.libphonenumber:libphonenumber:${libPhoneNumberVersion}"))
-      }
-    }
+		implementation.configure { impl ->
+			impl.withDependencies {
+				it.add(project.dependencies.create("com.googlecode.libphonenumber:libphonenumber:${libPhoneNumberVersion}"))
+			}
+		}
 
-    if (extension.generationMode != GenerationMode.API) {
-      if (extension.targetFramework == SPRING_BOOT) {
-        implementation.configure { impl ->
-          impl.withDependencies {
-            it.add(project.dependencies.create("ch.silviowangler.rest:rest-api-spring:${pluginVersion}"))
-          }
-        }
-        compileOnly.configure { cO ->
-          cO.withDependencies {
-            it.add(project.dependencies.create("org.springframework:spring-web:${springVersion}"))
-            it.add(project.dependencies.create("org.springframework:spring-webmvc:${springVersion}"))
-          }
-        }
-      } else if (extension.targetFramework.isMicronaut()) {
-        implementation.configure { impl ->
-          impl.withDependencies {
-            it.add(project.dependencies.create("ch.silviowangler.rest:rest-api-micronaut:${pluginVersion}"))
-          }
-        }
-      }
-    }
-  }
+		if (extension.generationMode != GenerationMode.API) {
+			if (extension.targetFramework == SPRING_BOOT) {
+				implementation.configure { impl ->
+					impl.withDependencies {
+						it.add(project.dependencies.create("ch.silviowangler.rest:rest-api-spring:${pluginVersion}"))
+					}
+				}
+				compileOnly.configure { cO ->
+					cO.withDependencies {
+						it.add(project.dependencies.create("org.springframework:spring-web:${springVersion}"))
+						it.add(project.dependencies.create("org.springframework:spring-webmvc:${springVersion}"))
+					}
+				}
+			} else if (extension.targetFramework.isMicronaut()) {
+				implementation.configure { impl ->
+					impl.withDependencies {
+						it.add(project.dependencies.create("ch.silviowangler.rest:rest-api-micronaut:${pluginVersion}"))
+					}
+				}
+			}
+		}
+	}
 }

--- a/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/RestApiPlugin.groovy
+++ b/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/RestApiPlugin.groovy
@@ -23,7 +23,6 @@
  */
 package ch.silviowangler.gradle.restapi
 
-
 import ch.silviowangler.gradle.restapi.tasks.CleanRestApiTask
 import ch.silviowangler.gradle.restapi.tasks.ExtractRestApiSpecsTask
 import ch.silviowangler.gradle.restapi.tasks.GenerateRestApiAsciiDocTask
@@ -32,13 +31,10 @@ import ch.silviowangler.gradle.restapi.tasks.PlantUmlTask
 import ch.silviowangler.gradle.restapi.tasks.ValidationTask
 import org.gradle.api.Plugin
 import org.gradle.api.Project
-import org.gradle.api.Task
 import org.gradle.api.artifacts.Configuration
-import org.gradle.api.tasks.compile.JavaCompile
 
 import static ch.silviowangler.gradle.restapi.Consts.CONFIGURATION_REST_API
 import static ch.silviowangler.gradle.restapi.Consts.TASK_GROUP_REST_API
-import static ch.silviowangler.gradle.restapi.TargetFramework.MICRONAUT
 import static ch.silviowangler.gradle.restapi.TargetFramework.SPRING_BOOT
 
 /**
@@ -46,94 +42,116 @@ import static ch.silviowangler.gradle.restapi.TargetFramework.SPRING_BOOT
  */
 class RestApiPlugin implements Plugin<Project> {
 
-	public static final String PLUGIN_ID = 'ch.silviowangler.restapi'
+  public static final String PLUGIN_ID = 'ch.silviowangler.restapi'
 
 
-	void apply(Project project) {
+  void apply(Project project) {
 
-		project.apply(plugin: 'java-library')
+    project.apply(plugin: 'java-library')
 
-		RestApiExtension extension = new RestApiExtension(project)
-		project.extensions.add('restApi', extension)
+    RestApiExtension extension = new RestApiExtension(project)
+    project.extensions.add('restApi', extension)
 
 
-		def clean = project.tasks.register('cleanRestArtifacts', CleanRestApiTask) { CleanRestApiTask t ->
-			t.group = TASK_GROUP_REST_API
-		}
-		def extract = project.tasks.register('extractSpecs', ExtractRestApiSpecsTask) { ExtractRestApiSpecsTask t ->
-			t.group = TASK_GROUP_REST_API
-		}
-		def validate = project.tasks.register('validateRestSpecs', ValidationTask) { ValidationTask t ->
-			t.group = TASK_GROUP_REST_API
-			if (ExtractRestApiSpecsTask.isConfigurationRestApiDefined(project)) {
-				t.dependsOn(extract)
-			}
-		}
-		def generate = project.tasks.register('generateRestArtifacts', GenerateRestApiTask) { GenerateRestApiTask t ->
-			t.group = TASK_GROUP_REST_API
-			t.dependsOn(validate)
-		}
-		def generateDiagrams = project.tasks.register('generateDiagrams', PlantUmlTask) { PlantUmlTask t ->
-			t.group = TASK_GROUP_REST_API
-		}
-		def generateAsciiDocs = project.tasks.register('generateAsciiDocs', GenerateRestApiAsciiDocTask) { GenerateRestApiAsciiDocTask t ->
-			t.group = TASK_GROUP_REST_API
-		}
+    def clean = project.tasks.register('cleanRestArtifacts', CleanRestApiTask) { CleanRestApiTask t ->
+      t.group = TASK_GROUP_REST_API
+    }
+    def extract = project.tasks.register('extractSpecs', ExtractRestApiSpecsTask) { ExtractRestApiSpecsTask t ->
+      t.group = TASK_GROUP_REST_API
+    }
+    def validate = project.tasks.register('validateRestSpecs', ValidationTask) { ValidationTask t ->
+      t.group = TASK_GROUP_REST_API
+      if (ExtractRestApiSpecsTask.isConfigurationRestApiDefined(project)) {
+        t.dependsOn(extract)
+      }
+    }
+    def generate = project.tasks.register('generateRestArtifacts', GenerateRestApiTask) { GenerateRestApiTask t ->
+      t.group = TASK_GROUP_REST_API
+      t.dependsOn(validate)
+    }
+    def generateDiagrams = project.tasks.register('generateDiagrams', PlantUmlTask) { PlantUmlTask t ->
+      t.group = TASK_GROUP_REST_API
+    }
+    def generateAsciiDocs = project.tasks.register('generateAsciiDocs', GenerateRestApiAsciiDocTask) { GenerateRestApiAsciiDocTask t ->
+      t.group = TASK_GROUP_REST_API
+    }
 
-		project.tasks.named('clean').configure {
-			dependsOn(clean)
-		}
+    project.tasks.named('clean').configure {
+      dependsOn(clean)
+    }
 
-		project.tasks.named('compileJava').configure {
-			dependsOn(generate)
-			options.encoding = 'UTF-8'
-		}
+    project.tasks.named('compileJava').configure {
+      dependsOn(generate)
+      options.encoding = 'UTF-8'
+    }
 
-		project.tasks.named('compileTestJava').configure {
-			options.encoding = 'UTF-8'
-		}
+    project.tasks.named('compileTestJava').configure {
+      options.encoding = 'UTF-8'
+    }
 
-		project.sourceSets.main.java.srcDir { project.restApi.generatorOutput }
+    project.sourceSets.main.java.srcDir { project.restApi.generatorOutput }
 
-		Configuration restApiSpecification = project.configurations.findByName(CONFIGURATION_REST_API)
+    Configuration restApiSpecification = project.configurations.findByName(CONFIGURATION_REST_API)
 
-		if (!restApiSpecification) {
-			project.configurations.create(CONFIGURATION_REST_API)
-		}
+    if (!restApiSpecification) {
+      project.configurations.create(CONFIGURATION_REST_API)
+    }
 
-		final String springVersion = "5.2.4.RELEASE"
-		final String pluginVersion = "2.3.14"
-		final String libPhoneNumberVersion = "8.11.5"
+    final String springVersion = "5.2.4.RELEASE"
+    final String pluginVersion = "2.3.15"
+    final String libPhoneNumberVersion = "8.11.5"
 
-		project.afterEvaluate {
+    final List<String> deps = [
+        "javax.annotation:javax.annotation-api:1.3.2",
+        "ch.silviowangler.rest:rest-model:${pluginVersion}",
+        "javax.money:money-api:1.0.3",
+        "javax.validation:validation-api:2.0.1.Final"
+    ]
 
-			project.dependencies {
+    def api = project.configurations.named("api")
+    def implementation = project.configurations.named("implementation")
+    def compileOnly = project.configurations.named("compileOnly")
 
-				if (extension.generationMode.isApiCodeGenerationRequired() || extension.generationMode.isClientCodeGenerationRequired()) {
-					api "javax.annotation:javax.annotation-api:1.3.2"
-					api "ch.silviowangler.rest:rest-model:${pluginVersion}"
-					api "javax.money:money-api:1.0.3"
-					api "javax.validation:validation-api:2.0.1.Final"
-				}
-				else {
-					implementation "javax.annotation:javax.annotation-api:1.3.2"
-					implementation "ch.silviowangler.rest:rest-model:${pluginVersion}"
-					implementation "javax.money:money-api:1.0.3"
-					implementation "javax.validation:validation-api:2.0.1.Final"
-				}
-				implementation "com.googlecode.libphonenumber:libphonenumber:${libPhoneNumberVersion}"
+    if (extension.generationMode.isApiCodeGenerationRequired() || extension.generationMode.isClientCodeGenerationRequired()) {
+      api.configure { a ->
+        a.withDependencies {
+          deps.each { dep -> it.add(project.dependencies.create(dep)) }
+        }
+      }
+    } else {
+      implementation.configure { impl ->
+        impl.withDependencies {
+          deps.each { dep -> it.add(project.dependencies.create(dep)) }
+        }
+      }
+    }
 
-				if (extension.generationMode != GenerationMode.API) {
+    implementation.configure { impl ->
+      impl.withDependencies {
+        it.add(project.dependencies.create("com.googlecode.libphonenumber:libphonenumber:${libPhoneNumberVersion}"))
+      }
+    }
 
-					if (extension.targetFramework == SPRING_BOOT) {
-						implementation "ch.silviowangler.rest:rest-api-spring:${pluginVersion}"
-						compileOnly "org.springframework:spring-web:${springVersion}"
-						compileOnly "org.springframework:spring-webmvc:${springVersion}"
-					} else if (extension.targetFramework.isMicronaut()) {
-						implementation "ch.silviowangler.rest:rest-api-micronaut:${pluginVersion}"
-					}
-				}
-			}
-		}
-	}
+    if (extension.generationMode != GenerationMode.API) {
+      if (extension.targetFramework == SPRING_BOOT) {
+        implementation.configure { impl ->
+          impl.withDependencies {
+            it.add(project.dependencies.create("ch.silviowangler.rest:rest-api-spring:${pluginVersion}"))
+          }
+        }
+        compileOnly.configure { cO ->
+          cO.withDependencies {
+            it.add(project.dependencies.create("org.springframework:spring-web:${springVersion}"))
+            it.add(project.dependencies.create("org.springframework:spring-webmvc:${springVersion}"))
+          }
+        }
+      } else if (extension.targetFramework.isMicronaut()) {
+        implementation.configure { impl ->
+          impl.withDependencies {
+            it.add(project.dependencies.create("ch.silviowangler.rest:rest-api-micronaut:${pluginVersion}"))
+          }
+        }
+      }
+    }
+  }
 }

--- a/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/builder/jaxrs/JaxRsRootResourceFactory.groovy
+++ b/gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/builder/jaxrs/JaxRsRootResourceFactory.groovy
@@ -108,12 +108,12 @@ class JaxRsRootResourceFactory extends AbstractResourceBuilder {
 	@Override
 	List<AnnotationSpec> getQueryParamAnnotations(VerbParameter paramName) {
 		// TODO handle VerbParameter options like required
-		return Collections.singletonList(createAnnotation(JAX_RS_QUERY_PARAM, ['value': paramName.getName()]))
+		return Collections.singletonList(createAnnotation(JAX_RS_QUERY_PARAM, ['value': paramName.name]))
 	}
 
 	@Override
 	List<AnnotationSpec> getHeaderAnnotations(Header header) {
-		return Collections.singletonList(createAnnotation(JAX_RS_HEADER_PARAM, ['value': header.getName()]))
+		return Collections.singletonList(createAnnotation(JAX_RS_HEADER_PARAM, ['value': header.name]))
 	}
 
 	@Override
@@ -139,7 +139,7 @@ class JaxRsRootResourceFactory extends AbstractResourceBuilder {
 		specs << createProducesAnnotation(representation)
 
 		if (isIdGenerationRequired(methodContext)) {
-			String postfix = representation.isJson() ? '' : ".${representation.name}"
+			String postfix = representation.json ? '' : ".${representation.name}"
 			specs << createAnnotation(JAX_RS_PATH, ['value': "{id}${postfix}"])
 		}
 		return specs

--- a/rest-api-micronaut/build.gradle
+++ b/rest-api-micronaut/build.gradle
@@ -1,7 +1,7 @@
 ext { pomDescription = 'Add ons for the Micronaut framework' }
 
 repositories {
-  mavenCentral()
+	mavenCentral()
 }
 
 dependencies {

